paddlecloud/paddlejob/views.py

```diff
@@ -89,6 +91,23 @@ def post(self, request, format=None):
         # get user specified image
         job_image = obj.get("image", None)
         gpu_count = obj.get("gpu", 0)
+        # jobPackage validation: startwith /pfs
+        # NOTE: always overwrite the job package when the directory exists
+        job_package =obj.get("jobPackage", "")
```

Comment:
After a brief discussion, we decide to use "jobname" as default path to store the package content. So that users can find history package and program file folders if they want to find out something.